python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -5,6 +5,13 @@
 import numpy as np
 from scipy import interpolate as ipol
 
+import astropy
```

Comment:
``astropy`` sub-modules are imported explicitly later on. Is it necessary to import the top-level astropy package?